src/Model.cc

```diff
@@ -33,7 +33,8 @@ class ignition::gazebo::ModelPrivate
   public: Entity id{kNullEntity};
 };
 
-using namespace ignition::gazebo;
+using namespace ignition;
+using namespace gazebo;
```

Comment:
Will anything this file includes ever be in the `gazebo` namespace? If so, it looks like splitting the using declaration would cause a compile error.

```
#include <iostream>

namespace gazebo
{
  void say_hello() { std::cout << "Hello from gazebo::say_hi()\n";}
}

namespace ignition
{
namespace gazebo
{
  void say_hi() { std::cout << "Hi from ignition::gazebo::say_hi()\n";}
}
}


// using namespace ignition::gazebo;
using namespace ignition;
using namespace gazebo;

int main()
{

  say_hello();
  say_hi();
  return 0;
}
```

```console
$ g++ -std=c++14 test.cpp
test.cpp:19:17: error: reference to ‘gazebo’ is ambiguous
 using namespace gazebo;
[...]
```

Reply:
I don't think that's a concern, we don't use any `gazebo` namespace outside of `ignition`.